test/gtest/graphapi_conv_bias_res_add_activ_fwd.cpp

```diff
@@ -308,86 +309,183 @@ class GPU_ConvBiasResAddActivation_fwd
+                                                             .build()));
+
+        ASSERT_NO_THROW(
+            graphBuilder.addNode(allocator.allocate(gr::OperationConvolutionForwardBuilder()
+                                                        .setConvolution(convolution)
+                                                        .setX(convInput)
+                                                        .setY(convOutput)
+                                                        .setW(convWeight)
+                                                        .setAlpha(1.0)
+                                                        .setBeta(0)
+                                                        .build())));
+
+        auto addInput =
+            allocator.allocate(gr::makeTensor<false>(addInputName, dataType, allOnes, allOnes));
+        auto addOutput =
+            allocator.allocate(gr::makeTensor<true>(addOutputName, dataType, allOnes, allOnes));
+
+        ASSERT_NO_THROW(graphBuilder.addNode(
+            MakeAddNode(allocator, dataType, convOutput, addInput, addOutput, alpha1, alpha2)));
+
+        miopen::TensorDescriptor biasInputTensorDesc{dataType, tensorLayout, allOnes};
+        auto biasInput =
+            allocator.allocate(gr::makeTensor<false>(biasInputName, dataType, allOnes, allOnes));
+        auto biasOutput =
+            allocator.allocate(gr::makeTensor<true>(biasOutputName, dataType, allOnes, allOnes));
+
+        ASSERT_NO_THROW(graphBuilder.addNode(
+            MakeAddNode(allocator, dataType, addOutput, biasInput, biasOutput)));
+
+        auto activationOutput = allocator.allocate(
+            gr::makeTensor<false>(activationOutputName, dataType, allOnes, allOnes));
+
+        gr::Pointwise* activation = nullptr;
+        ASSERT_NO_THROW(activation = allocator.allocate(gr::PointwiseBuilder{}
+                                                            .setMode(MIOPEN_POINTWISE_RELU_FWD)
+                                                            .setMathPrecision(dataType)
+                                                            .build()));
+        ASSERT_NO_THROW(graphBuilder.addNode(allocator.allocate(gr::OperationPointwiseBuilder{}
+                                                                    .setPointwise(activation)
+                                                                    .setX(biasOutput)
+                                                                    .setY(activationOutput)
+                                                                    .build())));
+
+        auto& handle   = get_handle();
+        auto handlePtr = static_cast<miopenHandle_t>(&handle);
+        graphBuilder.setHandle(handlePtr);
+        gr::OpGraph graph;
+        ASSERT_NO_THROW(graph = std::move(graphBuilder).build());
+        std::vector<gr::Engine> engines;
+        ASSERT_THROW(engines = gr::findEngines(&graph), miopen::Exception);
+    }
```

Comment:
That's a good point, ill try to combine these.

Thanks!

Reply:
Oh right I remember now, the reason I can't combine these is that it will throw earlier if I try to do the same flow with improperly formed arguments.

I think If I were to try and combine them, then the initialization would diverge quite a bit.